test/libyul/yulOptimizerTests/rematerialiser/no_remat_in_loop.yul

```diff
@@ -0,0 +1,17 @@
+{
+	let a := origin()
+	let b := calldataload(0)
+	let i := 0
+	for {} lt(i, 10) {i := add(a, b)} {
+	}
+}
+// ====
+// step: rematerialiser
+// ----
+// {
+//     let a := origin()
+//     let b := calldataload(0)
+//     let i := 0
+//     for { } lt(i, 10) { i := add(origin(), b) }
```

Comment:
Did `a` get rematerialized because `refs <=5 && cost <=1` is true above?

Reply:
I guess this is rather ``cost == 0``.